components/malloc_size_of/lib.rs

```diff
@@ -892,6 +892,10 @@ impl MallocSizeOf for xml5ever::QualName {
 malloc_size_of_is_0!(time::Duration);
 #[cfg(feature = "servo")]
 malloc_size_of_is_0!(time::Tm);
+#[cfg(feature = "servo")]
+malloc_size_of_is_0!(std::time::Duration);
+#[cfg(feature = "servo")]
+malloc_size_of_is_0!(std::time::SystemTime);
```

Comment:
I think servo feature is meant for external crates.

Reply:
I tried to use the same approach as `Tm` and `time::Duration `